report/2.Analysis/Requirements.tex

```diff
@@ -0,0 +1,39 @@
+\section{Requirements}
+In order to be able to conclude upon the success of the project, a fixed set of requirements must be formulated.
+These requirements will serve as a guideline in the conclusion of the report and as markers throughout the project. 
+To formally specify the requirements, the MoSCoW analysis model will be utilized. 
+The MoSCoW model specifies which requirements \textbf{M}ust, \textbf{S}hould, \textbf{C}ould, and \textbf{W}on't be fulfilled to deem the project successful.
+
+\textbf{Must have}
+\begin{itemize}
+\item ability to recognize a moving target in front of the device using an attached camera
```

Comment:
måske ændre denne så den er mere binær om den er løst.
"ability to detect whether a target is in front of the device"
camera er en del af løsningen - vel ikke en del af problemet.